hw/ip/uart/data/uart.hjson

```diff
@@ -140,13 +140,17 @@
           name: "RXIDLE"
           desc: "RX is idle"
           resval: "1",
-          tags: [// check idle in directed test only
-            "excl:CsrNonInitTests:CsrExclCheck"]
+          tags: [// check idle in other tests only. In top-level, RX pin is driven by 0 when it's
+                 // not selected at pinmux
+            "excl:CsrAllTests:CsrExclCheck"]
         }
         { bits: "5"
           name: "RXEMPTY"
           desc: "RX FIFO is empty"
           resval: "1"
+          tags: [// check idle in other tests only. In top-level, RX pin is driven by 0 when it's
+                 // not selected at pinmux
+            "excl:CsrAllTests:CsrExclCheck"]
```

Comment:
this change is due to top-level pinmux. need to exclude chekc for all the RX status related CSR.

@tjaychen it's because of the same reason you excluded `RXIDLE` recently. I also added exclusion for checking after reset, as reset seq may take a while and RX status may be changed due to RX pin=0.
If it's OK to you, can you approve this PR?

Reply:
sgtm, thanks @weicaiyang